src/big-int/bigint.hh

```diff
@@ -166,12 +168,11 @@ public:
   BigInt (llong_t) _fast;
   BigInt (ullong_t) _fast;
   BigInt (BigInt const &) _fast;
+  BigInt (BigInt &&) _fast;
   BigInt (char const *, onedig_t = 10) _fast;
 
-  BigInt &operator= (llong_t) _fast;
-  BigInt &operator= (ullong_t) _fast;
```

Comment:
In line with the recent update: shouldn't those also be kept (and be implemented via `swap`)?

Reply:
I don't think so. The best way to implement copy-assign safely is to set up another object with the desired state after assignment, and then to do a nothrow swap to exchange the states. That way, exceptions can only be thrown during the "setting up a new object" stage, which doesn't affect the logical state of the current object.

Reply:
With this in mind, there's no benefit between having explicit overloads for assignment from primitive types, and allowing BigInts to be implicitly constructed during assignment. We'll end up allocating a fresh temporary object in both cases.